images/kwok/build.sh

```diff
@@ -110,35 +120,52 @@ function dry_run() {
 
 function main() {
   local extra_args
+  local platform_args
+  local images
   local tag
 
   extra_args=()
+  images=()
   for image in "${IMAGES[@]}"; do
     tag="${VERSION}"
     if [[ "${STAGING_PREFIX}" != "" ]]; then
       tag="${STAGING_PREFIX}-${VERSION}"
     fi
-    extra_args+=(
-      "--tag=${image}:${tag}"
-    )
+    extra_args+=("--tag=${image}:${tag}")
+    images+=("${image}:${tag}")
+
     if [[ "${#EXTRA_TAGS[@]}" -ne 0 ]]; then
       for extra_tag in "${EXTRA_TAGS[@]}"; do
         tag="${extra_tag}"
         if [[ "${STAGING_PREFIX}" != "" ]]; then
           tag="${STAGING_PREFIX}-${extra_tag}"
         fi
-        extra_args+=(
-          "--tag=${image}:${tag}"
-        )
+        extra_args+=("--tag=${image}:${tag}")
+        images+=("${image}:${tag}")
       done
     fi
   done
 
   for platform in "${PLATFORMS[@]}"; do
-    extra_args+=(
-      "--platform=${platform}"
-    )
+    extra_args+=("--platform=${platform}")
+    platform_args+=("--platform=${platform}")
   done
+
+  if [[ "${BUILDER}" == "nerdctl" ]]; then
+    build_with_nerdctl "${extra_args[@]}"
+    if [[ "${PUSH}" == "true" ]]; then
+      for image in "${images[@]}"; do
+        dry_run nerdctl push "${platform_args[@]}" "${image}"
+      done
+    fi
+  else
+    build_with_docker "${extra_args[@]}"
```

Comment:
```suggestion
    if [[ "${PUSH}" == "true" ]]; then
      extra_args+=("--push")
    else
      extra_args+=("--load")
    fi
    build_with_docker "${extra_args[@]}"
```

So let's keep their styles consistent